lib/in_app_update.dart

```diff
@@ -78,6 +81,10 @@ class InAppUpdate {
     );
   }
 
+  static Stream<int> get installUpdateListener {
+    return _installListener.receiveBroadcastStream().cast<int>();
```

Comment:
@ViniciusSossela One last question: Is there an enum in Dart, to which the integers from Android map to?

Then the end user does not have to figure out which integer is which install status value.